.github/CONTRIBUTING.md

```diff
@@ -46,7 +46,7 @@ There is no need to squash your work as we will do that when the maintainer merg
   ```
   type(scope):desc
   ```
-  **Type** can be `fix`, `feat`, `docs`, `test` or `refactor`.
+  **Type** can be `chore`, `feat`, `docs`, or `test`.
```

Comment:
Could we keep `test` part of `chore`? Not sure how we can separate feature and breaking changes better than adding `breaking change`, or should we keep it in `feat`?

Reply:
@DalderupMaurice what do you think